RESOURCES.md

```diff
@@ -39,6 +41,16 @@
 37. [Oneview_volume_attachment](#oneview_volume_attachment)
 38. [Oneview_volume_template](#oneview_volume_template)
 
+## HPE Image Streamer resources
+
+1. [Image_streamer_artifact_bundle](#image_streamer_artifact_bundle)
+2. [Image_streamer_build_plan](#image_streamer_build_plan)
+3. [Image_streamer_deployment_group](#image_streamer_deployment_group)
+4. [Image_streamer_deployment_plan](#image_streamer_deployment_plan)
+5. [Image_streamer_golden_image](#image_streamer_golden_image)
+6. [Image_streamer_os_volume](#image_streamer_os_volume)
+7. [Image_streamer_plan_script](#image_streamer_plan_script)
+
 #### oneview_connection_template
```

Comment:
Missing a Title before the resources such as `## HPE OneView resources`, which should be linked by the summary item